config/locales/client.en.yml

```diff
@@ -40,20 +42,31 @@ en:
           private: Followers Only
           public: Public
         description:
-          private: ActivityPub Note is addressed to followers
-          public: ActivityPub Note is publicly addressed
+          private: "%{object_type} is addressed to followers."
+          public: "%{object_type} is publicly addressed."
       post_object_type:
         label:
           note: Note
           article: Article
         description:
           note: Best for microblogging platforms (e.g. Mastodon).
           article: Best for longform platforms (e.g. Lemmy).
+      publication_type:
+        label:
+          first_post: First Post
+          full_topic: Full Topic
+        description:
+          first_post: First post of every topic in this category is published via ActivityPub.
+          full_topic: Every post of every topic in this category is published via ActivityPub and replies on other ActivityPub servers are imported as replies.
     post:
       discourse_activity_pub:
         title:
-          published: "Post was published via ActivityPub at %{time}"
-          scheduled: "Post is scheduled to be published via ActivityPub at %{time}"
-          scheduled_past: "Post was scheduled to be published via ActivityPub at %{time}"
-          deleted: "ActivityPub note was deleted at %{time}"
-          updated: "Post was updated via ActivityPub at %{time}"
\ No newline at end of file
+          published: "%{object_type} was published from this site at %{time}."
+          scheduled: "%{object_type} is scheduled to be published from this site at %{time}."
+          scheduled_past: "%{object_type} was scheduled to be published from this site at %{time}."
+          deleted: "%{object_type} was deleted at %{time}."
+          updated: "%{object_type} was updated at %{time}."
+          published_remote: "%{object_type} was published from %{domain} at %{time}."
```

Comment:
I think we should still keep the `via ActivityPub` note in these messages, otherwise people may get confused here. (The strings are getting longer, but I'm not sure we can do anything about that.)

Reply:
Added back: https://github.com/angusmcleod/discourse-activity-pub/commit/1be6e59f038d9ede34be5013370c15d42c95595d